varcode/infer_effect.py

```diff
@@ -177,7 +177,7 @@ def infer_coding_effect(
     # In case sequence isn't a multiple of 3, then truncate it
     # TODO: if we get a frameshift by the end of a CDS (e.g. in the stop codon)
     # then we should use some of the 3' UTR to finish translation.
-    truncated_variant_cds_seq = variant_cds_seq[:len(variant_cds_seq) / 3 * 3]
+    truncated_variant_cds_seq = variant_cds_seq[:int(len(variant_cds_seq) / 3) * 3]
```

Comment:
i'm also going to add an "from __future__ import division" so we at least have the same semantics between 2 and 3 for these.